Reorganize codelab (#352)
transparency-dev/trillian-tessera: cmd/conformance/README.md

 - [A local filesystem](./posix/)
 - [MySQL](./mysql/)
 - [GCP](./gcp/)
 - [AWS](./aws/)

Each of these personalities exposes an endpoint that accepts `POST` requests at a `/add` URL.
The contents of any request body will be appended to the log, and the decimal index assigned to this newly _sequenced_ entry will be returned.

## Codelab

This codelab will help you add a few entries to a log, and read log contents.

> First, you need to bring up a Trillian Tessera log and to start personality (a server that 
> interfaces with the log) on the infrastructure of your choice:

Reviewer: ```suggestion
First, you need to bring up personality (a server built with Tessera which manages the log) on the infrastructure of your choice:
```

>  - [A local filesystem](./posix#bring-up-a-log)

Reviewer: ```suggestion
 - [A local POSIX-compliant filesystem](./posix#bring-up-a-log)
```
